specifyweb/specify/views.py

```diff
@@ -399,86 +400,156 @@ class ReplaceRecordPT(PermissionTarget):
     update = PermissionTargetAction()
     delete = PermissionTargetAction()
 
+@transaction.atomic
+def record_merge_fx(model_name: str, old_model_id: int, new_model_id: int) -> http.HttpResponse:
+    """Replaces all the foreign keys referencing the old record ID
+    with the new record ID, and deletes the old record record.
+    """
+    # Confirm the target model table exists
+    target_model = apps.get_model('specify', model_name.lower())
```

Comment:
This approach, while fine, is not consistent with getting models on the backend. 
The most common approach is to import `models` from `specifyweb.specify` and use `getattr(models, MODEL_NAME)`

Here are a few examples: https://github.com/specify/specify7/blob/fc70113acb58b91513af8d2a8025f742699cb36c/specifyweb/businessrules/uniqueness_rules.py#L2-L8
https://github.com/specify/specify7/blob/fc70113acb58b91513af8d2a8025f742699cb36c/specifyweb/accounts/views.py#L32
https://github.com/specify/specify7/blob/fc70113acb58b91513af8d2a8025f742699cb36c/specifyweb/workbench/tasks.py#L60

Reply:
It looks like you already have this imported. In that case, it would be `getattr(spmodels, model_name)`

On a side note, how does the frontend send the model_name? If it sends the model name in camel case, you would probably need to do `model_name.lower().capitalize()`